internal/runbits/promptable/promptable.go

```diff
@@ -0,0 +1,51 @@
+package promptable
+
+import (
+	"fmt"
+
+	"github.com/ActiveState/cli/internal/logging"
+	"github.com/ActiveState/cli/internal/output"
+)
+
+// IsPromptable reports whether the output type permits prompts to be shown.
+func IsPromptable(out output.Outputer) bool {
+	return out.Type() == output.PlainFormatName
```

Comment:
Output has nothing to do with this. Look into how we're reading `--non-interactive`.